boards/wio_terminal/src/wifi.rs

```diff
@@ -94,6 +94,9 @@ impl Wifi {
         })
     }
 
+    /// Turns on internal interrupts. Call this after you have finished
+    /// initializing the rest of your peripherals but before you start
+    /// issuing RPCs against the wifi chip.
     pub fn enable(&mut self, _cs: &CriticalSection, nvic: &mut NVIC) {
```

Comment:
Whelp, how are you meant to get access to it / enable interrupt lines?

Reply:
https://rtic.rs/0.5/book/en/internals/interrupt-configuration.html

Reply:
I think a doc comment will be sufficient.

Reply:
This can't really be used with RTIC anyway, because the interrupt handlers are defined in `wifi_singleton!` and not in the RTIC app macro.

Reply:
Are you aware of a peripheral that is laid out such that it can be used with RTIC? maybe i can follow that arrangement.

Alternatively, I can make sure we have a method that only enables the peripheral interrupts, and expect RTIC users to wire the priorities etc themselves without any macro help.

Reply:
I asked in the embedded channel a few days ago, didnt get any response. I'm going to leave this for now, I can always update it later if a better way becomes apparent.